Monika After Story/game/screens.rpy

```diff
@@ -833,7 +833,7 @@ screen game_menu(title, scroll=None):
 
         action Return()
 
-    label title
+    label title style ("game_menu_label" if not mas_globals.dark_mode else "game_menu_label_dark")
```

Comment:
branch checks are expensive when running screens, all new dark mode style adjustments should be done as a var in `mas_ui` that is changed when the dark mode switch is flipped. See the selector screen for examples.